Dockerfile.gaia

```diff
@@ -13,9 +13,21 @@ ENV GOOS=linux
 ENV GOFLAGS="-buildvcs=false"
 
 # fetch gaia from tag and build it
```

Comment:
Would also be ideal if we could pass a local build of gaia

Reply:
Agreed, @MSalopek or I could prob knock this out pretty quickly by just pattern matching the `-use-local-sdk` flag does things